juju/python-libjuju
# [JUJU-2401] Added release candidate workflow.

#### Description

An additional GitHub action triggered with cron to run integration tests with the latest available juju release candidate. 

#### QA Steps

Check that the observed juju release candidate version is the expected one. Rerun the job to check that most of the steps are simply skipped without a new version.

#### Notes & Discussion

NA
